client/src/components/Home/components/Editor.vue

```diff
@@ -47,12 +47,17 @@ h2 {
   font-weight: normal;
 }
 .editor {
-  background-color: burlywood;
-  width: 500px;
-  height: 500px;
+  background-color: #1e2127;
+  color: #9ba1a2;
+  width: 100%;
+  height: 100%;
   white-space: pre;
+  font-size: 16px;
+  text-align: left;
+  padding-left: 60px;
```

Comment:
We might want to use a `float: right` for the editor once the sidebar (with a `float: left`) is added.

`flex` or `table` row styles would work, as well. Maybe even more so, to make sure the appropriate line number is aligned with the line it refers to.